lib/avo/services/authorization_service.rb

```diff
@@ -68,12 +70,18 @@ def authorized_methods(user, record)
           end.to_h
         end
 
-        def get_policy(user, record)
-          Pundit.policy user, record
+        def get_policy(user, record, policy_class: nil)
```

Comment:
if we initialize the way i suggested this method can be deleted and wher it's called we can do something like `policy_class&.new(user, record)`